Sources/CallWithDoxDialer/CallWithDoxDialer.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+#if canImport(UIKit)
+import UIKit
+
+/// A struct to handle dialing phone numbers using the Doximity app.
+public struct DoxDialerCaller {
```

Comment:
Similar to the package name, if you want to follow the newer Ads SDK they made a product decision that we want to spell out `Doximity` as much as possible and get away from `Dox` in any public APIs. The types there are spelled out like https://github.com/doximity/ios-ads-sdk/blob/develop/DoximityAdsSDK/Sources/DoximityAdsPortal.swift if you want to follow it.